### DISCUSSION TITLE
Is there a shortcut link for a repository's latest Pre-release?

### DISCUSSION THREAD

**Discussion by lwcorp** [community/community#72604] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

As [documented here](https://docs.github.com/en/repositories/releasing-projects-on-github/linking-to-releases#linking-to-the-latest-release), the link `https://github.com/[username]/[repository]/releases/latest` automatically redirects into the latest release a repository has.

Is there a similar link that redirects to the latest **pre-release**?
e.g.
* `https://github.com/[username]/[repository]/releases/pre-latest`
* `https://github.com/[username]/[repository]/releases/latest-pre`
* `https://github.com/[username]/[repository]/releases/beta-latest`
* `https://github.com/[username]/[repository]/releases/latest-beta`

![image](https://github.com/community/community/assets/1773306/8b437b6e-cd20-456a-bf7c-ba04d6c5cfab)

**Comment by oussamaef06** [comment]
GitHub doesn't provide a built-in feature or URL structure that automatically redirects to the latest pre-release of a repository. The "latest" in the URL typically points to the latest stable release.

If you want to link to the latest pre-release, you may need to manually find and update the URL when a new pre-release is created. You can do this by visiting the "Releases" section of the repository and selecting the latest pre-release from the list.

Keep in mind that GitHub's features and URL structures may change over time, so it's possible that new features have been introduced since my last update. I recommend checking GitHub's official documentation or community forums for the most up-to-date information on this topic.

**Reply by lwcorp** [reply]
Is "/latest" even mentioned in any official docs? **Update:** [yes](https://github.com/orgs/community/discussions/72604#discussioncomment-14459741).

**Comment by Nadelio** [comment]
Yes `/latest` is mentioned [here](https://docs.github.com/en/repositories/releasing-projects-on-github/linking-to-releases)

**Reply by lwcorp** [reply]
Thanks! Updated the OP accordingly.

**Reply by Nadelio** [reply]
Np
